SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/SpatialTestMultiServerUnrealComponents/TestUnrealComponents.cpp

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "TestUnrealComponents.h"
+
+#include "Net/UnrealNetwork.h"
+
+UDataOnlyComponent::UDataOnlyComponent()
+{
+	PrimaryComponentTick.bCanEverTick = true;
+	SetIsReplicatedByDefault(true);
+}
+
+void UDataOnlyComponent::GetLifetimeReplicatedProps(TArray<FLifetimeProperty>& OutLifetimeProps) const
+{
+	Super::GetLifetimeReplicatedProps(OutLifetimeProps);
+
+	DOREPLIFETIME(ThisClass, DataReplicatedVar);
+}
+
+UDataAndHandoverComponent::UDataAndHandoverComponent()
+{
+	PrimaryComponentTick.bCanEverTick = true;
+	SetIsReplicatedByDefault(true);
+}
+
+void UDataAndHandoverComponent::GetLifetimeReplicatedProps(TArray<FLifetimeProperty>& OutLifetimeProps) const
+{
+	Super::GetLifetimeReplicatedProps(OutLifetimeProps);
+
+	DOREPLIFETIME(ThisClass, DataReplicatedVar);
```

Comment:
Interesting that we don't have to do this for the handover variables

Reply:
We should definitely reimplement handover as standard replicated variables with conditions.